src/extension/options-page/DashboardDialogs/Base.tsx

```diff
@@ -149,6 +149,7 @@ const useDashboardDialogWrapperStyles = makeStyles((theme) =>
             flexDirection: 'column',
             width: (props) => (props.size === 'small' ? 280 : 440),
             padding: (props) => (props.size === 'small' ? '40px 24px !important' : '40px 36px !important'),
+            margin: '0 auto',
```

Comment:
![image](https://user-images.githubusercontent.com/63733714/94910855-dbf9e800-04d7-11eb-93cd-6943db5d4d8b.png)

to

![image](https://user-images.githubusercontent.com/63733714/94910888-e916d700-04d7-11eb-9469-ddc8615922d4.png)

Reply:
if can, plz try use flex.

see https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
I know flex, but it seems not easy to do at a glance, I will check again.

Reply:
I tried to set its parent to a flex box, but it doesn't looks well:

![image](https://user-images.githubusercontent.com/63733714/94913552-08affe80-04dc-11eb-8bd8-af45d80c54ee.png)

![image](https://user-images.githubusercontent.com/63733714/94913722-5462a800-04dc-11eb-8b7c-c5d69ba62bbd.png)

I found that its parent has property `'position': 'fix'`, then I get the answer https://stackoverflow.com/a/29036585/4803321

> Absolutely positioned boxes are **taken out of the normal flow**.

Reply:
OK.